aiohue/util.py

```diff
@@ -227,3 +227,48 @@ def mac_from_bridge_id(bridge_id: str) -> str:
         bridge_id[14:16],
     ]
     return ":".join(parts)
+
+
+class LimitedList(list):
```

Comment:
This is part of Python as [`deque`](https://docs.python.org/3/library/collections.html#collections.deque)